deploy/kubeturbo/values.yaml

```diff
@@ -89,6 +89,10 @@ HANodeConfig:
 #   GitopsApps: false (default: true)
 #   GoMemLimit: true (default: false)
 
+# Dynamic logging level 
```

Comment:
`Logging level. Changing this value does not require restart of kubeturbo.`

Reply:
done